doc/learning/tutorial.html

```diff
@@ -740,7 +740,7 @@ <h2 id="oo">Object-Orientation</h2>
       </p>
       <ul>
         <li>Hidden fields, defined with <code>::</code>, which do not appear in generated JSON</li>
-        <li>The <code>super</code> keyword, which has its usual meaning</li>
+        <li>The <code>super</code> keyword is used to access and call functions on an object's parent.</li>
```

Comment:
Thanks, I agree we should clarify this, but let's change the phrasing slightly.

1) "access and call functions" -> "access fields". Methods are a special case of fields in Jsonnet. All fields can be accessed with super regardless of whether they are functions or not.
2) "object's parent" -> "base object". Parent is an ambiguous term (it can be understood as an object _containing_ another object as opposed to the parent in the inheritance hierarchy.

Reply:
Awesome, thank you for the feedback!